### PR TITLE
Adds U2F Support

### PR DESCRIPTION
**Pull request by mide**
Adds [U2F](https://en.wikipedia.org/wiki/Universal_2nd_Factor) support, allowing users to press the button on their device during the authentication process. 

This is based heavily on #25 by @toby-jn, with a few minor changes/additions:

* README documentation about U2F usage
* Python2 and Python3 compatibility
* Version bump from `0.0.12` to `0.0.13`
* Cleans up code to pass more Flake8 tests

----

From @toby-jn in #25:
> Add support for U2F (Yubikey) login. Some specifics of Google's U2F implementation requires some monkey patching.
Support optionally installed with pip install -e aws-google-auth[u2f], to avoid requiring dependency on python-u2flib-host.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -125,6 +129,9 @@ If you have more than one role available to you, you'll be prompted to
 choose the role from a list; otherwise, if your credentials are correct,
 you'll just see the AWS keys printed on stdout.
 
+If you're using Docker with a U2F security key (Yubikey or similar) you won't
```
Comment (tjbaker) [inline README.rst]: The sentence is awkward,  perhaps you meant to say `If you're using a U2F security key with your Google account you won't be ...`

Can you expand on what `Docker/Host isolation` mean?

File: aws_google_auth/__init__.py [hunk 2]
```diff
@@ -13,6 +13,14 @@
 from lxml import etree
 import configparser
 
+# In Python3, the libray 'urlparse' was renamed to 'urllib.parse'. For this to
```
Comment (tjbaker) [inline aws_google_auth/__init__.py]: s/libray/library

File: README.rst (hunk 1 above)
Comment (tjbaker) [inline README.rst]: This sentence is awkward,  perhaps you meant to say `If you're using a U2F security key with your Google account you won't be ...`

Can you expand on what `Docker/Host isolation` mean?

Comment (mide) [reply README.rst]: I agree on the sentence; that definitely could have been more clear. Changes coming shortly. 

Regarding the Docker/Host isolation, I'll put a small summary in the README.rst, but in more detail:
* The user will likely (not confirmed) need to set up USB passthrough to their VM (for Docker Machine / Docker Toolbox)
* The container will likely (not confirmed) need to have additional mounts and/or the privileged flag. 
* I have confirmed it _does not_ work as currently documented if you have a U2F key enabled on your Google Account and you run this using Docker.

**Pull request merged**